pkg/daemon/daemon.go

```diff
@@ -41,6 +41,7 @@ type Daemon struct {
 	RootMountPoint   *string
 	CustomMountPoint *string
 	nydusdThreadNum  int
+	Client           nydussdk.Interface
```

Comment:
Should this be serialized into the meta db?

Reply:
You're right, Client can't be serialized. Skip it in new version. Thanks!